graph/pull_v2_test.go

```diff
@@ -167,28 +167,4 @@ func TestValidateManifest(t *testing.T) {
 	if err == nil || !strings.HasPrefix(err.Error(), "image verification failed for digest") {
 		t.Fatal("expected validateManifest to fail with digest error")
 	}
-
-	// Manifest with no signature
-
-	expectedWholeFileDigest := "7ec3615a120efcdfc270e9c7ea4183330775a3e52a09e2efb194b9a7c18e5ff7"
```

Comment:
This particular test is for the `validateManifest` function. It's no longer possible to deserialize the manifest in order to call `validateManifest`. We could have a separate test to ensure that unmarshaling fails. I think such a test would belong under `distribution/manifest/schema1`. Would you like a PR to distribution to add a test there?